File: array/1450_number_of_students_doing_homework_at_a_given_time/1450_number_of_students_doing_homework_at_a_given_time.py
from typing import List

class Solution:
    def busyStudent(self, startTime: List[int], endTime: List[int], queryTime: int) -> int:
        return sum(a<=queryTime<=b for a,b in zip(startTime,endTime))


startTime = [1,2,3]
endTime = [3,2,7]
queryTime = 4
res = Solution().busyStudent(startTime, endTime, queryTime)
print(res)

File: array/1450_number_of_students_doing_homework_at_a_given_time/1450_number_of_students_doing_homework_at_a_given_time.go
package main 

import "fmt"

func busyStudent(startTime []int, endTime []int, queryTime int) int {
    length:=len(startTime)
    res:=0
    for i:=0;i<length;i++{
       if queryTime>=startTime[i] &&  queryTime <=endTime[i] {
           res++
       }
    }
    return res;
}

func main() {
	startTime := []int{1,2,3}
	endTime := []int{3,2,7}
	queryTime := 4
	res := busyStudent(startTime, endTime, queryTime)
	fmt.Println(res)
}